example_python_travis.yml

```
@@ -9,9 +9,13 @@ env:
     - DOCTR_VERSIONS_MENU="1"
     # Dependency files used to build the documentation (space separated)
     - DOCS_REQUIREMENTS="dev-requirements.txt requirements.txt"
+
     # Options to be passed to flake8 for package linting. Usually this is just
     # the package name but you can enable other flake8 options via this config
     - PYTHON_LINT_OPTIONS="my_package"
+    # Options to be passed to pytest for package test. Often, this will be left
+    # blank but can be used to enable custom pytest features
+    - PYTEST_ADDOPTS="-W"
```

Looks good, but probably best to leave it blank for when people copy/paste this:

```suggestion
    - PYTEST_ADDOPTS=""
```

```
$ pytest -W
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: argument -W/--pythonwarnings: expected one argument
```